app/controllers/unsubscriptions_controller.rb

```diff
@@ -26,6 +26,7 @@ def confirmed
                   "You have been unsubscribed from ‘#{@title}’"
                 else
                   "You have been unsubscribed"
+      description = "It can take up to an hour for this change to take effect."
                 end
       flash[:success] = message if unsubscribed
```

Comment:
I think we need to add something here to pass the description to `flash[:success]`. 

Perhaps replacing this line with something like:

```
flash[:success] = {
    message: message,
    description: description
}
```